barretenberg/cpp/src/barretenberg/ultra_honk/oink_prover.cpp

```diff
@@ -25,8 +25,15 @@ template <IsUltraFlavor Flavor> OinkProverOutput<Flavor> OinkProver<Flavor>::pro
     // Compute grand product(s) and commitments.
     execute_grand_product_computation_round();
 
+    // Generate relation separators alphas for sumcheck
```

Comment:
could probably make a separate function for this

Reply:
they are not only for sumcheck, they are for batching all subrelations into the big U(G)H relation either in sumcheck or combiner computation, cna you make the comment more specific?